src/main/java/org/italiangrid/storm/webdav/utils/IOExceptionHelper.java

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: 2014 Istituto Nazionale di Fisica Nucleare
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.italiangrid.storm.webdav.utils;
+
+import java.io.IOException;
+import org.italiangrid.storm.webdav.error.DiskQuotaExceeded;
+import org.italiangrid.storm.webdav.error.StoRMWebDAVError;
+
+public final class IOExceptionHelper {
+
+  public static final String DISK_QUOTA_EXCEEDED = "Disk quota exceeded";
+
+  private IOExceptionHelper() {}
+
+  public static StoRMWebDAVError getStoRMWebDAVError(IOException e) {
+    if (DISK_QUOTA_EXCEEDED.equals(e.getMessage())) {
+      return new DiskQuotaExceeded(e.getMessage(), e);
```

Comment:
```suggestion
      return new DiskQuotaExceeded(DISK_QUOTA_EXCEEDED, e);
```
?
E' necessario fare il check su una stringa o si puo' fare anche su un error code?

Reply:
Non penso ci sia altro modo di farlo perché quella stringa penso provenga dall'OS. Non è una cosa solidissima, ma questo metodo l'ho semplicemente spostato fuori da `StoRMFileResource` per usarlo anche in `DefaultFSStrategy`.
Ho comunque applicato la modifica così evitiamo una chiamata di funzione.